typescript/cli/src/commands/send.ts

```diff
@@ -52,6 +53,11 @@ const messageOptions: { [k: string]: Options } = {
     description: 'Skip wait for message to be delivered',
     default: false,
   },
+  messageBody: {
+    type: 'string',
+    description: 'Optional Message body',
+    default: 'Hello!',
+  },
```

Comment:
This should be added to the message command's builder line because the transfer command won't work with a message body. It may confuse people to see that option in there

Reply:
fixed!